
> futzz@1.0.0 test /home/cris/futzz
> clear && node --max-old-space-size=20000 -r esm tests/run.js "ufo" "multi"

[H[2J[3JMulti auto mode
undefined
Enumerated 8 possibilities.
{ jobStarted: { running: 1 }, remaining: 7 }
{ jobStarted: { running: 2 }, remaining: 6 }
{ jobStarted: { running: 3 }, remaining: 5 }
{ jobStarted: { running: 4 }, remaining: 4 }
{ jobStarted: { running: 5 }, remaining: 3 }
{ jobStarted: { running: 6 }, remaining: 2 }
{ jobStarted: { running: 7 }, remaining: 1 }
{ jobStarted: { running: 8 }, remaining: 0 }
{ jobCompleted: { running: 7 } }
(node:412339) UnhandledPromiseRejectionWarning: TypeError: startNextJob is not a function
    at startRun (/home/cris/futzz/tests/run.js:140:11)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async runMultiAuto (/home/cris/futzz/tests/run.js:132:5)
    at async start (/home/cris/futzz/tests/run.js:39:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:412339) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:412339) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
